### PR TITLE
Add smoketest

### PR DESCRIPTION
**Pull request by jdolitsky**
Try cloning a repo and check for README.md

### CODE REVIEW

**Comment by jdolitsky** [comment]
appears to be failing on rootless .. any pointers?

**Comment by erikaheidi** [comment]
Yeah! I have been working on this image's README and tried to create a practical example, however cloning didn't work, I was always getting permission denied errors. This was yesterday so I hoped to try some more today, but I see it might be a permission issue when trying to create a directory for cloning a repo. Maybe /home/non-root is not writable by this user?

What I was trying:

```shell
docker run -it -v ${pwd}:/home/nonroot --rm distroless.dev/git clone git@github.com:distroless/static.git
```

Output:

```
fatal: could not create work tree dir 'static': Permission denied

```

File: test.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o errtrace -o pipefail
+
+IMAGE_NAME=${IMAGE_NAME:-"distroless.dev/git"}
+CLONE_URL=${CLONE_URL:-"https://github.com/distroless/git.git"}
+
+rm -rf smoketest
+mkdir smoketest
```
Comment (erikaheidi) [inline test.sh:9]: I think you'll need to run a `chmod go+wrx smoketest` here to make it work. cc @amouat

**Comment by jdolitsky** [comment]
the chmod worked. Thanks!

**Comment by amouat** [comment]
Is this good to merge @jdolitsky?

File: test.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+set -o errexit -o nounset -o errtrace -o pipefail
+
+IMAGE_NAME=${IMAGE_NAME:-"distroless.dev/git"}
+CLONE_URL=${CLONE_URL:-"https://github.com/distroless/git.git"}
+
+CLONEDIR="$(mktemp -d)"
+chmod go+wrx "${CLONEDIR}"
+
+# TODO: re-enable this delete
```
Comment (amouat) [inline test.sh:11]: What's this about? I couldn't quite follow.

**Review by amouat — APPROVED** [review]
Please clarify or remove the comment in test.sh, but otherwise looks good.

File: test.sh (hunk 2 above)
Comment (jdolitsky) [reply test.sh:11]: after the clone, this directory appears to be un-deletable

**Pull request merged**